packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -30,6 +31,7 @@ const DEFAULT_FORMAT: Required<TableFormat> = {
     headerRowColor: '#ABABAB',
     tableBorderFormat: TableBorderFormat.DEFAULT,
     keepCellShade: false,
+    verticalAlign: 'top',

```

Comment:
We should not add this here. We should only apply this change when insert a new table. Here it is common code and will be used by all table scenarios.

Reply:
I mentioned the same about shading, but it also is in the default format.

What is the purpose of the default format constant if not to apply by default to all scenarios that do not explicitly have a format of their own? If default format should just be used for new inserted tables, I suggest we move it there in both models, however that change should be done separately.

Reply:
I think it makes sense to move this format to somewhere we start to apply table format, so VTable will be fully decoupled from the formats.

But given the code here is just working, we'd better not change it, to avoid regression. If we want to refactor it, that can be a separate PR.